packages/fxa-settings/src/components/Ready/index.tsx

```diff
@@ -81,9 +86,15 @@ const Ready = ({
       <section>
         <div className="error"></div>
         {isSignedIn ? (
-          <FtlMsg id="ready-use-service" vars={{ serviceName }}>
-            <p className="my-4 text-sm">{`You’re now ready to use ${serviceName}`}</p>
-          </FtlMsg>
+          serviceName ? (
```

Comment:
@millsoper I made a few changes to the `Ready`component with this PR, I'd like to get your eye on these. I think we should use a ternary here to ensure that the string with the default "account settings" is localized. If we instead set a default value for the variable, it won't get picked up and localized. Other service names passed in as props will be brand terms, so it should generally be okay if those aren't localized.

Reply:
This is a great catch and a great fix! Thank you!!